src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_signature.h

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include "pal_jni.h"
+
+#define SIGNATURE_VERIFICATION_ERROR -1
+
+int32_t AndroidCryptoNative_SignWithSignatureObject(JNIEnv* env, jobject signatureObject, jobject privateKey, const uint8_t* dgst, int32_t dgstlen, uint8_t* sig, int32_t* siglen);
```

Comment:
I believe @elinor-fung said something on a different PR, that I don't remember commenting on... but I believe that in the other shims the prefix is only used on PALEXPORT routines.  But maybe it also was (or is intermittently) on cross-file routines.  Almost certainly isn't on C-static functions

Reply:
I like using it on the cross-file routines. Makes it a little easier to know at a glance at the usage site if it's private to this file.